File: BPMOPMobile_VietNamAirLines/BPMOPMobile_VietNamAirLines/BPMOPMobile.Droid.Tablet/Class/MyCustomViewPager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Support.V4.View;
using Android.Util;
using Android.Views;
using Android.Widget;

namespace BPMOPMobile.Droid.Tablet.Class
{
    public class MyCustomViewPager : ViewPager
    {
        bool _flagTouchable = true;

        protected MyCustomViewPager(IntPtr javaReference, JniHandleOwnership transfer) : base(javaReference, transfer)
        {
            this._flagTouchable = true;
        }

        public MyCustomViewPager(Context context) : base(context)
        {
            this._flagTouchable = true;
        }

        public MyCustomViewPager(Context context, IAttributeSet attrs) : base(context, attrs)
        {
            this._flagTouchable = true;
        }

        public void SetViewPagerTouchable(bool _flagTouchable)
        {
            this._flagTouchable = _flagTouchable;
        }
        public override bool OnTouchEvent(MotionEvent e)
        {
            return this._flagTouchable && base.OnTouchEvent(e);
        }
        public override bool OnInterceptTouchEvent(MotionEvent ev)
        {
            return this._flagTouchable && base.OnInterceptTouchEvent(ev);
        }

    }
}

File: VNABPMOP/app/src/main/java/com/vuthao/bpmop/base/custom/MyCustomViewPager.java
package com.vuthao.bpmop.base.custom;

import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.viewpager.widget.ViewPager;

public class MyCustomViewPager extends ViewPager {
    private boolean enabled;

    public MyCustomViewPager(@NonNull Context context) {
        super(context);
        this.enabled = true;
    }

    public MyCustomViewPager(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        this.enabled = true;
    }

    public void setPagingEnabled(boolean enabled) {
        this.enabled = enabled;
    }

    @Override
    public boolean onTouchEvent(MotionEvent ev) {
        if (this.enabled) {
            return super.onTouchEvent(ev);
        }

        return false;
    }

    @Override
    public boolean onInterceptTouchEvent(MotionEvent ev) {
        if (this.enabled) {
            return super.onInterceptTouchEvent(ev);
        }

        return false;
    }
}
